end-to-end-test/remote/specs/core/comparisonTab.screenshot.spec.js

```diff
@@ -213,181 +213,214 @@ describe('results view comparison tab screenshot tests', function() {
-            goToUrlAndSetLocalStorage(
-                `${CBIOPORTAL_URL}/results/comparison?Z_SCORE_THRESHOLD=2.0&cancer_study_id=coadread_tcga_pub&cancer_study_list=coadread_tcga_pub&case_set_id=coadread_tcga_pub_nonhypermut&comparison_selectedGroups=%5B"KRAS"%2C"NRAS"%2C"BRAF"%5D&comparison_subtab=mutations&gene_list=KRAS%20NRAS%20BRAF&gene_set_choice=user-defined-list&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=coadread_tcga_pub_gistic&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=coadread_tcga_pub_mutations`
-            );
-            browser.waitForVisible(
-                'div[data-test="GroupComparisonMutationEnrichments"]',
-                10000
-            );
-            browser.moveToObject('body', 0, 0);
-            var res = browser.checkElement('.msk-tab:not(.hiddenByPosition)', {
-                hide: ['.qtip'],
-            });
-            assertScreenShotMatch(res);
-        });
-
-        it('results view comparison tab mutation enrichments tab patient mode', function() {
-            browser.execute(function() {
-                comparisonTab.store.setUsePatientLevelEnrichments(true);
-            });
-            browser.waitForVisible(
-                'div[data-test="GroupComparisonMutationEnrichments"]',
-                10000
-            );
-            browser.moveToObject('body', 0, 0);
-            var res = browser.checkElement('.msk-tab:not(.hiddenByPosition)', {
-                hide: ['.qtip'],
-            });
-            assertScreenShotMatch(res);
-        });
-
-        it('results view comparison tab mutation enrichments tab 20 genes with highest frequency in any group', function() {
-            browser.execute(function() {
-                comparisonTab.store.setUsePatientLevelEnrichments(false);
-            });
-            browser.waitForVisible(
-                'div[data-test="GroupComparisonMutationEnrichments"]',
-                10000
-            );
-            browser.click('[data-test="selectGenes"]');
-            var input = $('input[data-test=numberOfGenes]');
-            input.setValue('20\n');
-            browser.waitForEnabled('[data-test="addGenestoBarPlot"]', 10000);
-            browser.click('[data-test="addGenestoBarPlot"]');
-            browser.waitForVisible('div[data-test="GeneBarPlotDiv"]', 10000);
-            browser.moveToObject('body', 0, 0);
-            var res = browser.checkElement('div[data-test="GeneBarPlotDiv"]', {
-                hide: ['.qtip'],
-            });
-            assertScreenShotMatch(res);
-        });
-
-        it('results view comparison tab mutation enrichments tab gene box highest average frequency', function() {
-            browser.click('[data-test="selectGenes"]');
-            browser.execute(function() {
-                genesSelection.onGeneListOptionChange({
-                    label: 'Genes with highest average frequency',
+        it.omit(
```

Comment:
Well, since these are remote e2e-tests these depend on live-portal API. As such these tests can be created/reactivated when the whole feature is accepted. Or is there another way to make these tests against a different portal?

Reply:
oh okay, sounds good